From: Colin Ian King <colin.king@canonical.com>
Date: Sun, 20 Mar 2016 17:44:56 +0000
Subject: net/mlx4: remove unused array zero_gid[]
Patch-mainline: v4.6-rc1
Git-commit: 5a779c4feda5d52e0f19e48af71b75dd1032a5b8
References: bsc#966191 FATE#320230 bsc#966186 FATE#320228

zero_gid is not used, so remove this redundant array.

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Benjamin Poirier <bpoirier@suse.com>
---
 drivers/net/ethernet/mellanox/mlx4/mcg.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/net/ethernet/mellanox/mlx4/mcg.c
+++ b/drivers/net/ethernet/mellanox/mlx4/mcg.c
@@ -39,8 +39,6 @@
 
 #include "mlx4.h"
 
-static const u8 zero_gid[16];	/* automatically initialized to 0 */
-
 int mlx4_get_mgm_entry_size(struct mlx4_dev *dev)
 {
 	return 1 << dev->oper_log_mgm_entry_size;
